.github/workflows/ci.yml

```diff
@@ -2,9 +2,9 @@ name: CI
 
 on:
   push:
-    branches: [ main ]
+    branches: [ simplify-networking ]
```

Comment:
Does not this disable tests for main?

Reply:
you are right! good catch, I'll add it instead of replace it